README.md

```diff
@@ -9,26 +9,67 @@ A practical functional library for Javascript programmers.
 [![Gitter](https://badges.gitter.im/Join Chat.svg)](https://gitter.im/ramda/ramda?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
 
-Goals
------
+Why Ramda?
+----------
 
 <img src="http://ramda.jcphillipps.com/logo/ramdaFilled_200x235.png" 
-     width="200" height="235" align="left" hspace="12" />
-Using this library should feel as much like using Javascript as 
-possible.  Of course it's functional Javascript, but we're not 
-introducing lambda expressions in strings, we're not borrowing consed 
+     width="200" height="235" align="right" hspace="12" />
+
+Have you ever dreamed about snapping code together just like lego bricks ?
+You don't need to learn a functional programming language to reap the
+rewards of simpler code. Ramda makes it simple for you to build complex
+logic through functional composition (don't worry this only means sticking
+lego bricks together).
+
+Because of design decisions that you don't need to worry about either, Ramda
+can get your job done more elegantly, more flexible and with less code.
+
+Compare
+-------
+
+Let's compare the typical `LoDash` or `Underscore` chain style
+
+```js
+     var phonesSortedByName = function(users){
+          return _.chain(users)
+               .sortBy(function(user){
+                    return user.name;
+               })
```

Comment:
_.map(x, 'a') <=> _.map(x, _.property('a'))
On Jan 2, 2015 11:31 AM, "Scott Sauyet" <notifications@github.com> wrote:

> In README.md
> <https://github.com/ramda/ramda/pull/712#discussion-diff-22414398>:
>
> > +lego bricks together).
> > +
> > +Because of design decisions that you don't need to worry about either, Ramda
> > +can get your job done more elegantly, more flexible and with less code.
> > +
> > +Compare
> > +-------
> > +
> > +Let's compare the typical `LoDash` or `Underscore` chain style
> > +
> > +```js
> > +     var phonesSortedByName = function(users){
> > +          return _.chain(users)
> > +               .sortBy(function(user){
> > +                    return user.name;
> > +               })
>
> @megawac <https://github.com/megawac>: I don't want this for other
> reasons, but what do you mean by "implicit properties"?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ramda/ramda/pull/712/files#r22414398>.
>